src/test/munit/xa/activemq-xa-other-broker-test-case.xml

```diff
@@ -0,0 +1,235 @@
+                must be rolled back -->
+        <munit:execution>
+            <set-payload value="someValue"/>
+            <try>
+                <try transactionType="XA" transactionalAction="ALWAYS_BEGIN">
+                    <flow-ref name="publish-on-other-broker"/>
+                    <flow-ref name="publish-on-memory"/>
+                    <raise-error type="APP:SOME"/>
+                    <error-handler>
+                        <on-error-propagate/>
+                    </error-handler>
+                </try>
+                <error-handler>
+                    <on-error-continue/>
+                </error-handler>
+            </try>
+        </munit:execution>
+        <munit:validation>
+            <try>
+                <jms:consume config-ref="JMS_Config_XA_OTHER_BROKER" destination="someDest"/>
+                <munit-tools:fail message="Operation should have been rolled back"/>
+                <error-handler>
+                    <on-error-continue type="JMS:TIMEOUT"/>
+                </error-handler>
+            </try>
+            <try>
+                <jms:consume config-ref="JMS_Config_XA" destination="someOtherDest"/>
+                <munit-tools:fail message="Operation should have been rolled back"/>
+                <error-handler>
+                    <on-error-continue type="JMS:TIMEOUT"/>
+                </error-handler>
+            </try>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="commit-all">
+        <!--    When error happens after operations, and error is handled by an on-error-continue, operations
+                must be committed -->
+        <munit:execution>
+            <try>
+                <try transactionType="XA" transactionalAction="ALWAYS_BEGIN">
+                    <set-payload value="first"/>
+                    <flow-ref name="publish-on-other-broker"/>
+                    <set-payload value="second"/>
+                    <flow-ref name="publish-on-memory"/>
+                    <raise-error type="APP:SOME"/>
+                    <error-handler>
+                        <on-error-continue/>
+                    </error-handler>
+                </try>
+                <error-handler>
+                    <on-error-continue/>
+                </error-handler>
+            </try>
+        </munit:execution>
+        <munit:validation>
+            <jms:consume config-ref="JMS_Config_XA_OTHER_BROKER" destination="someDest"/>
+            <munit-tools:assert-that expression="#[payload]" is="#[MunitTools::equalTo('first')]"/>
+            <jms:consume config-ref="JMS_Config_XA" destination="someOtherDest"/>
+            <munit-tools:assert-that expression="#[payload]" is="#[MunitTools::equalTo('second')]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="second-operation-fails">
+        <!--    When error happens during an operation, and error is handled by an on-error-continue, the first
```

Comment:
I think that you can move these comments to the `<munit:test` as the `description` attribute